### ISSUE TITLE
Application Management during Sleep Mode

### ISSUE THREAD

**Issue by OtterWays** [nobelization/panoramax-mobile-app#100]
When the application goes to sleep or goes to the background while taking photos, the camera freezes and the only solution is to restart the application. It should be possible to resume capture immediately while keeping the same sequence.

When you completely close the application before sending the photos to an instance, all photos are lost. Photos should be stored locally to be uploaded later.

When you put the application to sleep, in the background, or close it while uploading photos, the upload stops and the unsent photos are lost. The upload should either:
- pause and resume when the application returns to the foreground
- continue in the background

**Comment by mnalis** [comment]
Absolutely, that is main priority IMHO. Last time I went for some good half and hour of collecting panoramax track, and then I had incoming call, and when I answered and returned to the panoramax app, it was "hung" (stopped collecting pictures, but showing progress screen like the one between taking pictures), and all the pictures were lost as I could not do anything, and on app restart nothing was there  :cry:  

There is also an issue if anything happens (like bad internet connection) while the pictures are uploading - also the remaining pictures are lost.

**Reliability** (i.e. do not lose pictures ever, but allow to continue collecting and/or uploading if app was interrupted in any way) would be primary thing to fix for the non-beta release, IMHO. Otherwise it is very discouraging when you know every little interruption will lose all the work.

**Comment by chrisdebian** [comment]
I definitely support this as a very high priority ("Must Have"). I have stopped using the app to capture images, because of this problem.

Chris
